run

```diff
@@ -40,6 +39,7 @@ elif ! [ -n "${SSL_CERTS-}" ]; then
   cat /tmp/server-cert.pem /tmp/server-key.pem > /etc/ssl/cert.pem
   rm /tmp/server-*.pem
 fi
+SSL_CERTS="/etc/ssl/cert.pem"
```

Comment:
Can we move this to the `case $key` section above? i.e. instead of `SSL_CERTS="yes"` `SSL_CERTS="/etc/ssl/cert.pem"` so that we aren't overloading the usage of the variable?